api/v3/Activity/Getmonthswithactivities.php

```diff
@@ -0,0 +1,71 @@
+<?php
+use CRM_Civicase_ExtensionUtil as E;
+
+/**
+ * Activity.Getmonthswithactivities API specification
+ *
+ * @param array $spec description of fields supported by this API call
+ * 
+ * @return void
+ * @see http://wiki.civicrm.org/confluence/display/CRMDOC/API+Architecture+Standards
+ */
+function _civicrm_api3_activity_Getmonthswithactivities_spec(&$spec) {
+  $allowed = ['activity_date_time', 'activity_status_id', 'case_id'];
```

Comment:
Why are we only allowing 3 params? It was never mentioned in the requirements.
The API should allow all parameters as `Activity.get`

Reply:
Endpoint now allows all of `Activity.get`'s parameters